bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -442,7 +442,15 @@ private String getMergeReadyComment(String commitMessage, List<Review> reviews)
             }
             message.append("pushed to the `");
             message.append(pr.targetRef());
-            message.append("` branch. ");
+            message.append("` branch:\n\n");
+            divergingCommits.stream()
+                            .limit(10)
```

Comment:
Will this be the most recent or the oldest 10 diverging commits?

Reply:
This should give the 10 most recent.

Reply:
Sounds good :+1: